app/src/components/CityMap.tsx

```diff
@@ -5,9 +5,16 @@ import { geoJSONBoundingBox } from "@/util/geojson";
 import { Box, Center, Spinner } from "@chakra-ui/react";
 import type { GeoJsonObject } from "geojson";
 import { LatLngBoundsLiteral, LatLngExpression } from "leaflet";
+import L from "leaflet";
```

Comment:
```suggestion
import type L from "leaflet";
```

This means it's only used for the TS compiler but not included in the final page code.